.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 # Other generated files
 htmlcov
 .coverage
+py/desiutil/.cache
```

Comment:
How is this file or directory generated?  Remember that Python packages tend to get installed read-only, so this is not necessarily good practice.